Sources/ResearchKitSwiftUI/ORKOrderedTaskView.swift

```diff
@@ -76,6 +80,10 @@ public struct ORKOrderedTaskView: UIViewControllerRepresentable {
             shouldConfirmCancel
         }
 
+        public func taskViewControllerSupportsSaveAndRestore(_ taskViewController: ORKTaskViewController) -> Bool {
+            false // TODO: support that eventually?
```

Comment:
Fine if we don't support this right now. We could do this using a questionnaire response. Could be a cool first issue for someone. Can you add a GitHub issue for that?

Reply:
I will create a GitHub issue for that later 👍